include/boost/math/special_functions/detail/bernoulli_details.hpp

```diff
@@ -184,17 +184,22 @@ struct fixed_vector : private std::allocator<T>
    const T& operator[](unsigned n)const { BOOST_MATH_ASSERT(n < m_used); return m_data[n]; }
    unsigned size()const { return m_used; }
    unsigned size() { return m_used; }
-   void resize(unsigned n, const T& val)
+   bool resize(unsigned n, const T& val)
    {
       if(n > m_capacity)
       {
+#ifndef BOOST_NO_EXCEPTIONS
```

Comment:
I'm not an expert, but just curious here... Why is this unexpected event handled differently than the ones below that go through _policies_? Is there no such fitting policy for this action?